ascon/src/lib.rs

```diff
@@ -28,7 +30,7 @@ const fn round_constant(round: u64) -> u64 {
 /// The state of Ascon's permutation.
 ///
 /// The permutation operates on a state of 320 bits represented as 5 64 bit words.
-#[derive(Clone, Copy, Debug, Default)]
+#[derive(Clone, Debug, Default)]
```

Comment:
This breaks semver compatibility. Please bump the version `0.4.0-pre`.

Reply:
I updated the version accordingly.